### DISCUSSION TITLE
FoxESS current energy status graphic

### DISCUSSION THREAD

**Discussion by CandCB** [nathanmarlor/foxess_modbus#387] [Q&A] [answered]
Hi Nathan, all,
This might already be done, or just daft but here goes anyway - I'm fairly new to HA, very new to the FoxESS modbus integration and know next to nothing about Python, so not a great combination. However, I'm really impressed with this integration, much more stable than the native Fox cloud, faster update and more useful to drill down into the data.. Also, really like the Energy dashboard upgrade.
My question / suggestion is for the "Energy Distribution Today" graphic - I think it would be really nice to have the same style graphic but with current values, rather than daily cumulative, that can be displayed on user generated dashboards. I've tried to see where the code is for the existing card but can't find anything so might just not be looking in the right place. I'm hoping that you think this would be a good addition and can find time to include in a future release.
![FoxESS card](https://github.com/nathanmarlor/foxess_modbus/assets/138257729/343b6bd8-7e28-4de7-bb01-8b74ae02258d)
Thanks for your great work.
Colin

**Comment by canton7** [comment]
You might be after something like this card: https://github.com/ulic75/power-flow-card?

This integration doesn't provide any of the graphics on the Energy dashboard (it doesn't provide any graphics at all): it just gets data from the inverter and makes it available to other things in HA.

**Reply by CandCB** [reply]
Hi Canton7,
Looks to be pretty much what I'm after, not quite a flash as Nathan's graphic but should do the job for me. The water and gas sections won't apply to me but it looks as though the card is editable. I'll have a play - thanks,
Colin

**Reply by canton7** [reply]
Again, this integration does not provide any graphics. You're looking at something provided by home assistant.

**Reply by nathanmarlor** [reply, marked as answer]
> You might be after something like this card: https://github.com/ulic75/power-flow-card?
> 
> This integration doesn't provide any of the graphics on the Energy dashboard (it doesn't provide any graphics at all): it just gets data from the inverter and makes it available to other things in HA. 

Not sure why this exists in parallel but this version is much more configurable -> https://github.com/flixlix/power-flow-card-plus

**Comment by CandCB** [comment]
Thank you @canton7 & @nathanmarlor, I've now had chance to have try these out and the power-flow-card-plus is exactly what I was after.
Much appreciated.
